<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="channelGroupBox">
     <property name="title">
      <string>CH X</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Stim mode</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="stimModeComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Pulse amplitude</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="DelayScienDSpinBox" name="stimAmplitudeSpinBox"/>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="isBipolarCheckBox">
        <property name="toolTip">
         <string>In bipolar mode, the pulse is symetrical relative to 0</string>
        </property>
        <property name="text">
         <string>Bipolar</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Number of pulses</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="stimNPulsesSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Train Frequency</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="DelayScienDSpinBox" name="stimTrainFreqSpinBox">
        <property name="suffix">
         <string>Hz</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Train Duration</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="ScienDSpinBox" name="stimTrainDurationSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string>s</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="trigStimPushButton">
           <property name="text">
            <string>Trigger!</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/icons/Lightning</normalon>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="DelayScienDSpinBox" name="stimDurationSpinBox">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000001000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Pulse Duration</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="thresholdButton">
        <property name="toolTip">
         <string>Clicking this button enables &quot;Threshold&quot; mode, where the amplitude is defined as a multiple of the threshold value</string>
        </property>
        <property name="text">
         <string>threshold</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="DelayScienDSpinBox" name="thresholdValueSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScienDSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">src.scientific_spinbox</header>
  </customwidget>
  <customwidget>
   <class>DelayScienDSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">src.GUI</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>stimModeComboBox</tabstop>
  <tabstop>stimAmplitudeSpinBox</tabstop>
  <tabstop>thresholdButton</tabstop>
  <tabstop>thresholdValueSpinBox</tabstop>
  <tabstop>stimDurationSpinBox</tabstop>
  <tabstop>isBipolarCheckBox</tabstop>
  <tabstop>stimNPulsesSpinBox</tabstop>
  <tabstop>stimTrainFreqSpinBox</tabstop>
  <tabstop>stimTrainDurationSpinBox</tabstop>
  <tabstop>trigStimPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
